docs/src/snap/howto/storage/cloud.md

```diff
@@ -0,0 +1,476 @@
+# How to use cloud storage
+
+{{product}} simplifies the process of integrating and managing cloud storage
+solutions like Amazon EBS. This guide provides steps to configure IAM policies,
+deploy the cloud controller manager, and set up the necessary drivers for you
+to take advantage of cloud storage solutions in the context of Kubernetes.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to an Amazon EC2 instance
+- You can create roles and policies in AWS
+
+
```

Comment:
I suggest we say this is explicitly for AWS. In future we can use tabs to add Azure etc